src/System.Net.Security/src/System/Net/SecureProtocols/_SslState.cs

```diff
@@ -828,34 +780,25 @@ private void StartSendBlob(byte[] incoming, int count, AsyncProtocolRequest asyn
                     _Framing = DetectFraming(message.Payload, message.Payload.Length);
                 }
 
-                // Even if we are comleted, there could be a blob for sending.
-                // ONLY for TlsStream we want to delay it if the underlined stream is a NetworkStream that is subject to Nagle algorithm
-                // CONSIDER: Always buffer the last handshake payload and then flush it on a subsequent write (in worst case on read/flush/close)
-                if (message.Done && _ForceBufferingLastHandshakePayload && InnerStream.GetType() == typeof(NetworkStream) && !_PendingReHandshake)
```

Comment:
Right: The NetworkStream dependency was required by an optimization for TlsStream. That type is an internal part of the WebRequest stack which we're not porting.